SimpleBrowser/Browser.cs

```diff
@@ -289,11 +289,19 @@ public void ClearException()
 			LastWebException = null;
 		}
 
-		public void ClearWindows()
+		public void ClearWindowsInContext()
 		{
             foreach (var window in _allWindows.ToArray()) window.Close();
 			_allWindows.Clear();

```

Comment:
Be sure to add the removal of both ClearWindows () otherwise we will keep all links _allContexts, which will lead to memory leaks.